src/components/ThemeProvider/ThemeProvider.style.ts

```diff
@@ -0,0 +1,51 @@
+import { SerializedStyles, css } from '@emotion/react';
+import { normalize } from 'polished';
+import { Theme } from 'theme';
+
+export const scrollbar = (theme: Theme): SerializedStyles => css`
+  // for Firefox
+  * {
+    scrollbar-width: thin;
+  }
```

Comment:
This property is applied to **every** element in the DOM. Are we sure we need this?

Reply:
@kostasdano

Reply:
We need this since we need ALL scrollbars to inherit this style; on top of that we tested this version on products and everything runs smoothly so this doesn't create any issues